Spigot-Server-Patches/0399-Don-t-sleep-after-profile-lookups-if-not-needed.patch

```diff
@@ -7,30 +7,30 @@ Mojang was sleeping even if we had no more requests to go after
 the current one finished, resulting in 100ms lost per profile lookup
 
 diff --git a/src/main/java/com/mojang/authlib/yggdrasil/YggdrasilGameProfileRepository.java b/src/main/java/com/mojang/authlib/yggdrasil/YggdrasilGameProfileRepository.java
-index 71e48e87b..23f1447cf 100644
+index 26a743722..6ed3199c3 100644
 --- a/src/main/java/com/mojang/authlib/yggdrasil/YggdrasilGameProfileRepository.java
 +++ b/src/main/java/com/mojang/authlib/yggdrasil/YggdrasilGameProfileRepository.java
```

Comment:
Why does this file have a diff? It doesn't seem like anything changed 😐

The actual PR looks good to me 😀

Reply:
Probably because of the way how Paper scripts are generating patches?

Reply:
whitespace changes in the surrounding of a patch. 
view diffs with ?w=1 to hide those on github ;)

Reply:
There's also the \r character in there somewhere.

Reply:
Ah, thanks 😀